src/tests/ftest/util/fault_config_utils.py

```diff
@@ -235,6 +236,99 @@
         'probability_y': '100',
         'interval': '1',
         'max_faults': '1'},
+    'DLCK_FAULT_CREATE_LOG_DIR': {
+        'id': '131328',
+        'probability_x': '100',
+        'probability_y': '100',
+        'interval': '1',
+        'max_faults': '1'},
+    'DLCK_FAULT_CREATE_POOL_DIR': {
+        'id': '131329',
+        'probability_x': '100',
+        'probability_y': '100',
+        'interval': '1',
+        'max_faults': '1'},
+    'DLCK_FAULT_ENGINE_START': {
+        'id': '131330',
+        'probability_x': '100',
+        'probability_y': '100',
+        'interval': '1',
+        'max_faults': '1'},
+    'DLCK_FAULT_ENGINE_EXEC': {
+        'id': '131331',
+        'probability_x': '100',
+        'probability_y': '100',
+        'interval': '1',
+        'max_faults': '1'},
+    'DLCK_FAULT_ENGINE_JOIN': {
+        'id': '131332',
+        'probability_x': '100',
+        'probability_y': '100',
+        'interval': '1',
+        'max_faults': '1'},
+    'DLCK_FAULT_ENGINE_STOP': {
+        'id': '131333',
+        'probability_x': '100',
+        'probability_y': '100',
+        'interval': '1',
+        'max_faults': '1'},
+    'DAOS_FAULT_POOL_NVME_HEALTH': {
+        'id': '131584',
+        'probability_x': '100',
+        'probability_y': '100',
+        'interval': '2'},
+    'DAOS_FAULT_POOL_OPEN_BIO': {
+        'id': '131585',
+        'probability_x': '100',
+        'probability_y': '100'},
+    'DAOS_FAULT_POOL_OPEN_UMEM': {
+        'id': '131586',
+        'probability_x': '100',
+        'probability_y': '100',
+        'interval': '2'},
+    'DAOS_FAULT_POOL_OPEN_MAGIC': {
+        'id': '131587',
+        'probability_x': '100',
+        'probability_y': '100',
+        'interval': '2'},
+    'DAOS_FAULT_POOL_OPEN_VERSION': {
+        'id': '131588',
+        'probability_x': '100',
+        'probability_y': '100',
+        'interval': '2'},
+    'DAOS_FAULT_POOL_OPEN_UUID': {
+        'id': '131589',
+        'probability_x': '100',
+        'probability_y': '100',
+        'interval': '2'},
+    'DAOS_FAULT_BTREE_OPEN_INV_CLASS': {
+        'id': '131590',
+        'probability_x': '100',
+        'probability_y': '100',
+        'interval': '28',
+        'max_faults': '1'},
```

Comment:
This fault can be fine-tuned to hit either the containers tree (interval = 28) or the gc tree (interval = 29). Hence it seems we need two records with distinctive keys. I am sorry it was not obvious from the `fault_injection_dlck.yaml` file.

And IMHO we really need to give comments here explaining where 28 and 29 come from. Otherwise you won't reverse-engineer their meaning and considering these values are fine-tuned it could be crucial to adjust them in the future.

```suggestion
    'DAOS_FAULT_BTREE_OPEN_INV_CLASS_28': {
        'id': '131590',
        'probability_x': '100',
        'probability_y': '100',
        'interval': '28', # containers tree fine-tuned
        'max_faults': '1'},
    'DAOS_FAULT_BTREE_OPEN_INV_CLASS_29': {
        'id': '131590',
        'probability_x': '100',
        'probability_y': '100',
        'interval': '29', # gc tree fine-tuned
        'max_faults': '1'},
```